### PR TITLE
Reorganize

### PR DESCRIPTION
**Pull request by kathrynljackson**
## Description

- A Navbar has been added. I began formatting.
- Separated fetch requests into separate files
- Deleted unused `Deck` files
- `componentDidMount` function in `App.js` changed to use the  `fetchCards()` function in `fetchRequests.js`.

## Motivation and Context

I made this change in order to have buttons that Navigate to different parts of the app. I attempted to match the wireframe. 

## How Has This Been Tested?

The app still runs as it did before. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "bootstrap": "^4.5.3",
+    "node-sass": "^4.14.1",
```
Comment (CalebCyphers) [inline package.json:10]: Ooh, I'm excited for SASS

File: src/App/App.js [hunk 2]
```diff
@@ -2,18 +2,18 @@ import React, { Component } from 'react';
 import './App.css';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import { Container } from 'react-bootstrap'
-import Deck from '../Deck.js';
 import Loader from '../Loader/Loader.js'
 import DailyReading from '../DailyReading/DailyReading.js'
+import NavBar from '../NavBar/NavBar.js'
+import { fetchCards } from '../fetch/fetch-requests';
```
Comment (CalebCyphers) [inline src/App/App.js:8]: Thanks for working on getting the Navbar going, I'm excited!

File: src/fetch/fetch-requests.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export const fetchCards = () => {
+    return fetch('https://empress-tarot-api.herokuapp.com/cards')
+    .then(res => res.json())
+}
```
Comment (CalebCyphers) [inline src/fetch/fetch-requests.js:4]: noice

**Review by CalebCyphers — APPROVED** [review]
These changes look good. I think this will go a long way towards a nice, clean codebase

File: src/App/App.js (hunk 2 above)
Comment (chadrick-d-dev) [reply src/App/App.js:8]: Great executive decision to make the nav bar! Kathryn, you're awesome!

File: src/fetch/fetch-requests.js (hunk 3 above)
Comment (chadrick-d-dev) [reply src/fetch/fetch-requests.js:4]: It's done! Yes!

File: package.json (hunk 1 above)
Comment (chadrick-d-dev) [reply package.json:10]: I second that feeling!

**Pull request merged**